packages/cli/src/webpack/utils.ts

```diff
@@ -226,7 +226,7 @@ export function requireImaConfig(rootDir = process.cwd()): ImaConfig | null {
  * @param {ImaCliArgs} args CLI args.
  * @returns {Promise<ImaConfig>} Ima config or empty object.
  */
-export async function resolveImaConfig(args: ImaCliArgs): Promise<ImaConfig> {
+export function resolveImaConfig(args: ImaCliArgs): ImaConfig {
```

Comment:
If we cannot / should not make this function async, then we have to rethink `@ima/testing-library` method `render` to be either also async (and we will have to call `await render` in tests), or we have to separate context (or more specifically language) initialisation process into individual method and by this, remove the option to call `render` without any additional setup.

Reply:
It is better to be prepared for async future, these changes have been reverted and `@ima/testing-library` is more async friendly now.